src/base/timer.h

```diff
@@ -88,6 +88,8 @@ class Timer {
   struct timezone time_zone_;
 };
 
+#endif
+
 class Profiler {
  public:
```

Comment:
These changes seem unrelated to the original issue about class Profiler. I think the fix is to simply add `{}` after
`~Profiler()`

Reply:
Oh, it seems in the original file, the class Profiler will only be enabled under the Unix settings -- that is, when `if defined(_MSC_VER) || defined(MINGW)` (line 26) is false (starting from line 58).
Can we just add add `{}` after `~Profiler()`?

Reply:
If just add `{}` after `~Profiler()`, there will be no class definition for class Profiler in windows settings

Reply:
Oh sorry, I see it now, your fix might be right.
@btiplitz  can you please test?

Reply:
I’ll try it in the morning